### ISSUE TITLE
一些小问题

### ISSUE THREAD

**Issue by HoracceFeng** [Sierkinhane/crnn_chinese_characters_rec#170]
@Sierkinhane 你好，感谢你这套 pytorch 版的 CRNN，有两个小问题：

1. torch.nn.CTCLoss 跟原版 warpctc 相比有多大的性能差距？torch 版 CTCLoss 是否功能不全？tensorflow 版本中有参数可以实现输入 CTC 时字串长度与 conv 不同的操作。torch 版 API 没有，这一块是需要自己写是吗？

2. crnn_main.py: train() 函数每个 epoch 的 loss 都重新设置，是否对训练有影响？尤其是训练样本较少时（比如只有几万）
